composeApp/src/commonMain/kotlin/com/chukchukhaksa/moblie/domain/openmajor/usecase/GetOpenMajorListUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.chukchukhaksa.moblie.domain.openmajor.usecase
+
+
+import com.chukchukhaksa.moblie.domain.openmajor.repository.OpenMajorRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+class GetOpenMajorListUseCase (private val openMajorRepository: OpenMajorRepository) {
+    operator fun invoke(): Flow<List<String>> = flow {
+        emit(listOf("전체", "컴퓨터SW"))
+    }
+}
```

Comment:
Q. openMajorRepository 구현체가 없어서 DI가 안되지 않나요?
```suggestion
class GetOpenMajorListUseCase () {
    operator fun invoke(): Flow<List<String>> = flow {
        emit(listOf("전체", "컴퓨터SW"))
    }
}
```

이렇게만 바꿔도 지금은 충분할거같아요

Reply:
UseCase의 인자를 제거하게 되면 koin 모듈 설정하는 부분에서 get() 함수를 제거 해야 하기 때문에 추가를 했었습니다.

그럼 지금은 제거를 하고 repository 구현체 작업 완료 되면 그때 koin 설정 또한 진행하도록 하겠습니다.

Reply:
넵! 그렇게 부탁드릴게요!